### DISCUSSION TITLE
How to get handle of MainWindow in Avalonia

### DISCUSSION THREAD

**Discussion by NikhilTambi86** [AvaloniaUI/Avalonia#5905] [Q&A] [answered]
I need to get a handle of MainWindow in Avalonia framework.

    public static IntPtr MainWindow_handle = IntPtr.Zero;
    MainWindow MainWindow_ui = new MainWindow();
    MainWindow_handle = new WindowInteropHelper(MainWindow_ui).Handle;

Can somebody please answer on how to get handle of MainWinow?

**Comment by kekekeks** [comment, marked as answer]
`window.PlatformImpl.Handle.Handle`
